pkg/odo/util/cmdutils.go

```diff
@@ -83,6 +84,20 @@ func PrintComponentInfo(currentComponentName string, componentDesc component.Des
 		}
 
 	}
+	// Linked services
+	if len(componentDesc.LinkedServices) > 0 {
+		fmt.Print("Linked Services: ")
+		fmt.Printf("%v\n", strings.Join(componentDesc.LinkedServices, ","))
+	}
+	// Linked components
+	if len(componentDesc.LinkedComponents) > 0 {
+		fmt.Println("\nLinked Components")
```

Comment:
This isn't symetrical: if there are no linked services but linked components, there will be an extra `\n`, whereas in the opposite case, there won't…

Reply:
That is true, but the reason is that the output for linked components spans multiple lines. So I add the extra line break at the beginning to demarcate that components.
Of course when there are both linked components and services the services won't have a break after the components, so I will go ahead and add that

Reply:
Scratch that, services are printed before components, so I believe this is good the way it is

Reply:
This is what I mean regarding the extra `\n`:
- linked component, no service:
```Component Name: frontend
Type: nodejs
Source: file:///Users/claprun/Downloads

Linked Components
Name: backend - Port(s): 8080```

- linked service, no component:
```Component Name: backend
Type: python
Source: file:///Users/claprun/Downloads
Linked Services: mongodb-ephemeral```

Reply:
Also, it'd make sense for services and components to be displayed similarly.

Reply:
Linked components needs to display both the name and port - for services there is no such need so I don't that they should be displayed in the same way

Reply:
> Linked components needs to display both the name and port - for services there is no such need so I don't that they should be displayed in the same way

I know but what I meant is that components are all output on their own lines, while services are inlined. There's also the `:`. All minor points… :)

Reply:
I just tried to follow what was already being done with existing output :P